### PR TITLE
Initial Okteto stack

### PR DESCRIPTION
**Pull request by rberrelleza**
Initial version of the Okteto stack installer. It installs the private-cloud of https://cloud.okteto.com. Still putting the final touches on the post-install configuration UI, but wanted to file the PR early in case there are any comments or blockers.

### CODE REVIEW

**Comment by paulanunda** [comment]
Thanks for the PR @rberrelleza; I'll be reviewing it shortly.

File: stacks/okteto/deploy-local.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# deploy okteto
+kubectl apply -f "$ROOT_DIR"/stacks/okteto/yaml/crds.yaml
+kubectl apply -f "$ROOT_DIR"/stacks/okteto/yaml/okteto.yaml
+
+# generate a random password
+PASSWORD=$(head -c 12 /dev/urandom | shasum| cut -d' ' -f1)
+kubectl label sa do-okteto-enterprise dev.okteto.com/token=$PASSWORD --overwrite
+
+# ensure services are running
+kubectl rollout status deployment/do-okteto-enterprise-api
```
Comment (paulanunda) [inline stacks/okteto/deploy-local.sh]: It looks like a number of deployments are created. Does it make sense to monitor the rollout status for all of them?

```
NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
do-cainjector                           1/1     1            1           22m
do-cert-manager                         1/1     1            1           22m
do-nginx-ingress-controller             2/2     2            2           22m
do-nginx-ingress-default-backend        2/2     2            2           22m
do-okteto-enterprise-api                2/2     2            2           22m
do-okteto-enterprise-frontend           2/2     2            2           22m
do-okteto-enterprise-mutation-webhook   2/2     2            2           22m
do-webhook                              1/1     1            1           22m
```

**Comment by wadenick** [comment]
Hello again old friend @rberrelleza! While @paulanunda reviews the PR, I wanted to check in with you on readiness of the listing. Danny had shown me a few questions and I'm curious if you've worked through those as yet, and if the listing will be ready to release as soon as the PR is merged? 

Cheers, -nick

**Comment by rberrelleza** [comment]
> Hello again old friend @rberrelleza! While @paulanunda reviews the PR, I wanted to check in with you on readiness of the listing. Danny had shown me a few questions and I'm curious if you've worked through those as yet, and if the listing will be ready to release as soon as the PR is merged?
> 
> Cheers, -nick

Hey friend!  Yes, having the configuration UI (the part that's missing from this PR) fixed the issues I asked Danny about yesterday. Once this PR is ready we'll be good to go on the listing.

**Comment by rberrelleza** [comment]
@paulanunda made the change to wait for all the deployments to be ready. The PR is ready to be merged now.

**Pull request merged**